compiler/frontend/build.gradle.kts

```diff
@@ -6,6 +6,7 @@ plugins {
 dependencies {
     compile(project(":kotlin-annotations-jvm"))
     compile(project(":core:descriptors"))
+    compile(project(":core:descriptors.jvm"))
```

Comment:
I think this dependency should not be here

Reply:
+1! Please be careful with adding such dependencies. Common compiler code shouldn't depend on platform-specific parts. It sometimes really turns out quite difficult to get rid of such dependencies afterwards.

Reply:
As per my suggestion above, this dependency is removed at the last commit in this PR.

Reply:
Looks like your suggestion was about the `fir2ir` -> `fir:java` dependency, which is indeed removed and it's great :). But this one is the `frontend` -> `descriptors.jvm` dependency.

Reply:
Oh, there is another dependency addition, wasn't aware of that, sorry!